### PR TITLE
Update blocklistConfig.json

### PR DESCRIPTION
**Pull request by shuvashish76**
useful for Avast, AVG apps built in Android phones (for ex samsung phones). & added Hola lists.
source: https://github.com/ftpmorph/ftprivacy

### CODE REVIEW

**Comment by shuvashish76** [comment]
hese include domains also not sure if the format should be hosts or domains
`"format": "domains",`
&
```
"group": "parentalcontrol",
"subg": "services",
```
??
please check their .txt files & change if necessary

File: blocklistConfig.json [hunk 1]
```diff
@@ -1556,6 +1556,24 @@
...
       "subg": "",
       "url": "https://raw.githubusercontent.com/jerryn70/GoodbyeAds/master/Extension/GoodbyeAds-Spotify-AdBlock.txt"
+    },  
+    {
+      "value": 173,
+      "vname": "AVG+Avast full (ftprivacy)",
+      "uname": "AAF",
+      "format": "hosts",
+      "group": "privacy",
+      "subg": "",
```
Comment (ignoramous) [inline blocklistConfig.json:1566]: `services`.

File: blocklistConfig.json [hunk 2]
```diff
@@ -1556,6 +1556,24 @@
...
+      "url": "https://raw.githubusercontent.com/ftpmorph/ftprivacy/master/blocklists/avg-avast-data-mining-full-block.txt"
+    },
+    {
+      "value": 174,
+      "vname": "Hola full (ftprivacy)",
+      "uname": "HLF",
+      "format": "hosts",
+      "group": "privacy",
+      "subg": "",
```
Comment (ignoramous) [inline blocklistConfig.json:1575]: `services`.

**Review by ignoramous — COMMENTED** [review]
Thanks, `subg` for these should be `services`?

File: blocklistConfig.json (hunk 1 above)
Comment (shuvashish76) [reply blocklistConfig.json:1566]: I think same as these are for total blocking of Avast, AVG & Hola domains & trackers.
`"subg": "services"`
& `"group": "privacy"` is right one its clearly not for `parentalcontrol`

File: blocklistConfig.json (hunk 2 above)
Comment (shuvashish76) [reply blocklistConfig.json]: I'm not a dev, idk what failed & why failed. I just changed `"subg": "services"` section. :(
please check-edit & fix if necessary

Comment (ignoramous) [reply blocklistConfig.json]: No biggie, we've changed the blocklist format from underneath your commits, and hence the failure. The fields `value` and `uname` have been removed / deprecated. I'll update this commit shortly.

**Comment by ignoramous** [comment]
https://github.com/serverless-dns/blocklists/pull/7

**Pull request closed without merging**